src/store/useTracksStore.ts

```diff
@@ -24,7 +24,7 @@ const useTracksStore = create<TracksState>((set, get) => ({
   totalTracks: 0,
   totalPages: 0,
   itemsPerPage: 8,
-  sort: { field: "title", direction: "asc" },
+  sort: { field: "title" as SortField, direction: "asc" as SortDirection },
```

Comment:
You can set SortField and SortDirection as enum, and then use it
```suggestion
  sort: { field: SortField.title, direction: SortDirection.asc },
```